enhancements/authentication/direct-oidc-configuration.md

```diff
@@ -0,0 +1,457 @@
+---
+title: direct-oidc-configuration
+authors:
+  - stlaz
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - deads2k
+  - sjenning
+  - tkashem
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - deads2k
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - deads2k
+creation-date: 2023-10-10
+last-updated: yyyy-mm-dd
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - "https://issues.redhat.com/browse/HOSTEDCP-1240"
+see-also:
+  - "/enhancements/authentication/direct-oidc-study/study-oidc-in-openshift.md"
+  - "https://github.com/kubernetes/enhancements/commits/master/keps/sig-auth/3331-structured-authentication-configuration"
+replaces:
+  - "/enhancements/authentication/unsupported-direct-use-of-oidc.md"
+superseded-by: []
+---
+
+# Direct OIDC Configuraition
+
+## Summary
+
+This enhancement describes how to allow configuring OIDC directly as the main
+source of identities in an OpenShift cluster.
+
+## Motivation
+
+In vanilla Kubernetes, it is possible to configure a remote OIDC provider as the
+source of identities. This has made it possible to create solutions that are based
+around it. However, these are incompatible with the current OpenShift authentication
+stack.
+
+We want to both enable the aforementioned solutions, and we want to make it possible
+for users to be able to use their OIDC providers' features to their full extent.
+
+### User Stories
+
+* I am a cluster administrator and I want to be able to use the full potential
```

Comment:
nit: To cover for multi-cluster in both managed and self hosted scenarios we usually define stories thinking of the following personas: cluster admin, cluster dev, cluster provider, cluster consumer